test/valid-data-struct.test.ts

```diff
@@ -4,6 +4,7 @@ describe("valid-data-struct", () => {
     // TODO: generics recursive
     it("simple-object", assertValidSchema("simple-object", "SimpleObject"));
 
+    it.only("object-function-expression", assertValidSchema("object-function-expression", "MyType"));
```

Comment:
Remove the only

Reply:
Omg, can't believe I forgot to remove that. Thanks for catching this.